File: backend/tour-design-api/src/main/java/com/tourdesign/platform/model/PackageHotelModel.java
package com.tourdesign.platform.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PackageHotelModel {
    private Long id;
    private Integer orderIndex;

    private TravelPackageModel travelPackage;
    private HotelModel hotel;
}

File: frontend/tour-design-web/src/app/models/package-hotel.model.ts
// src/app/models/package-hotel.model.ts
import { TravelPackageModel } from './travel-package.model';
import { HotelModel } from './hotel.model';

export interface PackageHotelModel {
  id: number;
  orderIndex?: number;
  travelPackage?: TravelPackageModel;
  hotel?: HotelModel;
}
